PWGLF/TableProducer/LFTreeCreatorNuclei.cxx

```diff
@@ -165,6 +176,7 @@ struct LfTreeCreatorNuclei {
         track.pt(),
         track.p(),
         track.eta(),
+        track.y(),
```

Comment:
I'm afraid I still don't get why you would like to save the `y` of a track, are you referring to the rapidity instead?